configure.ac

```diff
@@ -1773,8 +1773,8 @@
         if test "$LIBBPF" = "no"; then
             echo
             echo "   libbpf library and development headers not found but"
-            echo "   but needed to use eBPF code. It can be found in the"
-            echo "   Linux kernel tree under tools/lib/bpf"
+            echo "   but needed to use eBPF code. It can be found in at"
+            echo "   https://github.com/libbpf/libbpf"
```

Comment:
what builtin version by kernel/distributions is also enough? maybe at this here or at the docs

Reply:
We got this in the doc:
```
Suricata XDP code has been tested with 4.13.10 but 4.15 or later is necessary to use all
features like the CPU redirect map.
```